### DISCUSSION TITLE
Support Wifi Dongle / Firmware

### DISCUSSION THREAD

**Discussion by sim-san** [home-assistant/operating-system#1941] [Q&A]
Is there somewhere a list of supported wifi dongle or wifi dongle firmware ?

My current wifi dongle does not work. So I plan to buy a new one and want to make a good decision .

**Comment by agners** [comment]
No, unfortunately we don't have such a list. Probably your best chance is to look though closed issues/pull requests to see which drivers got added.

What kind of model do you have which isn't working? If it's supported by the upstream Linux kernel, it should be fairly straight forward to enable support for it.

**Reply by sim-san** [reply]
My device has been detected. I would like to create an access point with it. For this I use this addon: https://github.com/mattlongman/Hassio-Access-Point

Unfortunately I get the following error, which is described in this except:

https://github.com/mattlongman/Hassio-Access-Point/issues/29

**Reply by sim-san** [reply]
Solved in https://github.com/mattlongman/Hassio-Access-Point/issues/29#issuecomment-1141694142
